ui/src/app/settings/components/repos-list/repos-list.tsx

```diff
@@ -173,7 +175,9 @@ export class ReposList extends React.Component<
             case ConnectionMethod.HTTPS:
                 const httpsValues = params as NewHTTPSRepoParams;
                 return {
-                    url: (!httpsValues.url && 'Repository URL is required') || (this.credsTemplate && !this.isHTTPSUrl(httpsValues.url) && 'Not a valid HTTPS URL'),
+                    url:
+                        (!httpsValues.url && 'Repository URL is required') ||
+                        (this.credsTemplate && !this.isHTTPSUrl(httpsValues.url) && !httpsValues.enableOCI && 'Not a valid HTTPS URL'),
```

Comment:
Is it intended that URL shouldn't be validated if `enableOCI` is `true` and URL doesn't include `https` protocol?

Reply:
yes. that's because internal logic expects the OCI url not to have a protocol